project.clj

```diff
@@ -10,7 +10,9 @@
              "-server"]
   :dependencies [[org.clojure/clojure "1.7.0"]
                  [clj-time "0.9.0"] ; required due to bug in lein-ring
-                 [metosin/schema-tools "0.7.0"]
+                 [prismatic/schema "1.0.5"]
```

Comment:
This is the version that compojure-api wants.  It was getting replaced by an older version.